test/htmlreporter.spec.ts

```diff
@@ -182,6 +182,48 @@ describe("HTMLReporter", () => {
       });
     });
 
+    describe("includeStackTrace", () => {
+      it("should remove stack traces in failure messages if set to false", async () => {
+        const reporter = new HTMLReporter({
+          testData: mockedJestResponseSingleTestResult,
+          options: {
+            includeFailureMsg: true,
+            includeStackTrace: false,
+          },
+        });
+        const reportContent = await reporter.renderTestReportContent();
+        expect(reportContent).toBeDefined();
+        expect(reportContent!.toString().indexOf("at stack trace")).toBe(-1);
+      });
+      it.only("should keep stack trace in failure messages if set to true", async () => {
```

Comment:
`.only` probably blocks the rest of the tests in the suite here

Reply:
Perfect! I must have forgotten to remove it while testing 🙏